src/components/Tabs.js

```diff
@@ -205,4 +128,184 @@ class TabWithoutRadium extends React.Component {
 
 const Tab = Radium(TabWithoutRadium);
 
+class MenuTabs extends StandardTabs {
```

Comment:
Its also ok if inheritance is used when it makes sense. While I also tend to not favor it, it is important not to get too dogmatic and use tools when they make sense. 

If we do go this route, maybe just a comment explaining why this pattern gets used here would be helpful for future developers and avoid them copying it by default without understanding the consequences elsewhere.